lib/pf/Switch.pm

```diff
@@ -3130,7 +3130,7 @@ Takes into account the active/active clustering and centralized deauth
 
 =cut
 
-sub deauth_source_ip {
+sub orig_deauth_source_ip {
```

Comment:
`orig_deauth_source_ip` doesn't sound exactly right to me.

I would personally merge the two subroutines together or call that one `_deauth_source_ip` or another more meaningful name than orig_...